bench/wtperf/wtperf.c

```diff
@@ -142,21 +143,20 @@ map_key_to_table(CONFIG_OPTS *opts, uint64_t k)
 
 /*
  * Figure out and extend the size of the value string, used for growing
- * updates. We know that the value to be updated is in the threads value
- * scratch buffer.
+ * updates/modifies, delta is the value to be updated according to the thread
```

Comment:
This is two complete sentences. It should be `updates/modifies. Delta is the...`.

Reply:
corrected.